src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -70,7 +83,7 @@ public void Execute()
                 }, TaskScheduler.FromCurrentSynchronizationContext());
         }
 
-        internal static bool ShowTermsOfUseDialog()
+        internal static bool ShowTermsOfUseDialog(bool forPublishing, string additionalTerms)
```

Comment:
Yep, it's not used yet. For now we always show identical TOU document, but in future if we choose to show different contents, then this can come in handy.